attributes/default.rb

```diff
@@ -71,7 +71,8 @@
 case node['platform_family']
 when 'debian'
   default['nrpe']['install_method']    = 'package'
-  default['nrpe']['pid_file']          = '/var/run/nagios/nrpe.pid'
+  default['nrpe']['pid_dir']           = '/var/run/nagios/'
+  default['nrpe']['pid_file']          = node['nrpe']['pid_dir'] + 'nrpe.pid'
```

Comment:
In general you want to avoid an attribute that's built with another attribute. If someone sets the pid_dir there are many scenarios where pid_file won't be built correctly.